.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+[Delete any non-applicable sections, but we may ask for more information. Please reference the [SmartDeviceLink GitHub Best Practices](https://d83tozu1c8tt6.cloudfront.net/media/resources/SDL_GitHub_BestPractices.pdf) for further instructions on how to enter an issue.]
+
+### Bug Report
+[Summary]
+
+##### Reproduction Steps
+1. [Step 1]
+2. [Step 2]
+3. [Step 3]
+
+##### Expected Behavior
+[Some expected behavior]
+
+##### Observed Behavior
+[Some observed behavior]
+
+##### OS & Version Information
+* SDL Security iOS Version: [What version of the sdl_security_ios library has this bug been seen on]
+* SDL iOS Version: [What version of the sdl_ios library was being used]
+* Testing Against: [What you tested with to observe this behavior]
```

Comment:
The version of iOS on the device may be important as well

Reply:
Fixed. Added